EnglishTranslation-master/ETC.tsv

```diff
@@ -17624,7 +17624,7 @@ ETC_20151224_017625	There's no one who wants to see the true nature of the hidde
 ETC_20151224_017626	The phenomena of the Golemlet absorbing the energy from Tree Root Crystals to become stronger occurs sometimes.
 ETC_20151224_017627	Deadborns turn into Rodelins when they receive a lot of influences from demons or when they are acknowledged by the high-ranked demons.
 ETC_20151224_017628	Even when the Pantos were having a close relationship with humans, they actually weren't that friendly to humans. But, they weren't as hostile as they are now.
-ETC_20151224_017629	There's a legend which Ritheris was one of the masters in the past. Among the masters that are not wizard types, some masters make contracts with the demons for the eternity and look for others who are similar to them and lead them to the demon realm.
+ETC_20151224_017629	There's a legend which Riteris was one of the masters in the past. Among the masters that are not wizards, some masters make contracts with the demons for the eternity and look for others who are similar to them and lead them to the demon realm.
```

Comment:
`There's a legend where Riteris was one of the masters in the past. Among the masters that are not wizards, some make contracts with the demons for an eternity and look for others who are similar to them, then lead them to the demon realm.`